pkg/scheduler/framework/types.go

```diff
@@ -164,14 +184,25 @@ func (ce ClusterEvent) IsWildCard() bool {
 }
 
 // Match returns true if ClusterEvent is matched with the coming event.
+// "match" means the coming event is the same or more specific than the ce.
+// i.e., when ce.ActionType is Update, it return true if a coming event is UpdateNodeLabel
```

Comment:
OK, so this is the last part we may argue about, we have the power of interpretation but have no idea whether this will break users. cc @alculquicondor @Huang-Wei who have contextual knowledge.

Reply:
Maybe not matter because we already categoties in the front when event comes, like the node events.

Reply:
For a combinatorial event, e.g. podLabel & podRequest， it should match with each event I think.

Reply:
It shouldn't be a concern; Let's have an example with Pod/Update and Pod/PodLabelUpdate:
- if a user registers Pod/Update, it matches if an incoming event is Pod/PodLabelUpdate (actually any types of Pod update events).
- if a user registers Pod/PodLabelUpdate, it matches **_only_** if an incoming event is Pod/PodLabelUpdate. It doesn't match if an incoming event is Pod/Update.
  - Before this PR, it matches in the latter case too, which is unnecessary match.
    - Why unnecessary match? > because the scheduler interprets an event as Pod/PodLabelUpdate if a Pod got changed in labels. This means that if receiving a Pod/Update event, it's always an event that doesn't change the labels of the Pod.

Reply:
uhm.... maybe we need `UpdatePodOther` that gets added when no other `podChangeExtracters` find a change?

`comingEvent.ActionType <= ce.ActionType` just looks very unintuitive

Reply:
Implemented `UpdatePodOther` with a caution to users.